client/src/sass/include/_mixins.scss

```diff
@@ -636,6 +688,21 @@
   overflow: hidden;
   font-size: 0.75rem;
   border-radius: 0.25rem;
+  isolation: isolate;
+  position: relative;
+
+  span {
```

Comment:
What are the purposes of these rules?

Reply:
`isolation: isolate;` is not useful anymore.